command/issue_test.go

```diff
@@ -641,6 +643,78 @@ func TestIssueCreate_webTitleBody(t *testing.T) {
 	eq(t, output.String(), "Opening github.com/OWNER/REPO/issues/new in your browser.\n")
 }
 
+func TestIssueCreate_interactiveSubmitWithTemplate(t *testing.T) {
+	initBlankContext("", "OWNER/REPO", "master")
+	http := initFakeHTTP()
+	http.StubRepoResponse("OWNER", "REPO")
+
+	http.StubResponse(200, bytes.NewBufferString(`
+		{ "data": { "repository": {
+			"id": "REPOID",
+			"hasIssuesEnabled": true
+		} } }
+	`))
+	http.StubResponse(200, bytes.NewBufferString(`
+		{ "data": { "createIssue": { "issue": {
+			"URL": "https://github.com/OWNER/REPO/issues/12"
+		} } } }
+	`))
+
+	var templateFiles []string
+	if rootDir, err := git.ToplevelDir(); err == nil {
```

Comment:
My hope with template testing is that we're not reading from the filesystem. This can fail or change behavior as we change the repository itself.

I'd rather have a function wrapper around finding templates on the filesystem that can be overriden in tests to provide predictable results.

Reply:
Oh. Okay. I was actually thinking that since the code is intended to read from the filesystem and we are also reading from the filesystem in the tests, it should not be a problem. Since changes to the file system would affect the code and the tests in the same way, so they would always be in sync